components/core/src/error.rs

```diff
@@ -267,6 +269,11 @@ impl fmt::Display for Error {
                  is in the form application.environment (example: twitter.prod)",
                 e
             ),
+            Error::InvalidBinding(ref binding) => format!(
+                "Invalid binding \"{}\", must be of the form <NAME>:<SERVICE_GROUP> where <NAME> \
```

Comment:
```suggestion
                "Invalid binding '{}', must be of the form <NAME>:<SERVICE_GROUP> where <NAME> \
```
We use single quotes for internal quoting [elsewhere](https://github.com/habitat-sh/core/blob/master/components/core/src/error.rs#L175) in [this file](https://github.com/habitat-sh/core/blob/master/components/core/src/error.rs#L346-L347), and since I find that easier to read than the escapes, I suggest we go that way.